frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -66,14 +74,12 @@ export const AddCapacityModal = (props: AddCapacityModalProps) => {
     currentCapacity = (
       <div className="skeleton-text ceph-add-capacity__current-capacity--loading" />
     );
-  } else if (loadError || !cephCapacity || !osdSizeWithoutUnit || deviceSetIndex === -1) {
+  } else if (loadError || !osdSizeWithoutUnit) {
```

Comment:
If `values` come as an empty arrays of arrays when loaded is true and `loadError` is null.
Then `totalCapacity` and `usedCapacity` will be `undefined` hence add a check for them to show "Not available".
```suggestion
  } else if (loadError ||  !totalCapacity || !usedCapacity) {
```